### ISSUE TITLE
Define rule DDF00005

### ISSUE THREAD

**Issue by nickdedonder** [cdisc-org/ddf-core-poc#6]

**Comment by DianeWold** [comment]
Does the rule need to specify that studyTitle is an attribute of StudyVersion?

**Comment by DianeWold** [comment]
@rbakercdisc1 Could not find the rule in the rule editor.

**Comment by DianeWold** [comment]
@rbakercdisc1 The comment in the rule is, "#Study class has the studyTitle attribute".  However, Study doesn't have a title attribute.  I think this rule should probably be "StudyProtocolDocumentVersion has a scientificTitle"  StudyProtocolDocumentVersion has more than one "title" attribute, but I think this rule is about a scientificTitle, rather than a publicTitle or a briefTitle. 
Perhaps the mismatch is because I'm looking at the markdown Gerry created, and that is for a different version of the model than the one you were looking at.

**Comment by rbakercdisc1** [comment]
@DianeWold  Yes, it was another version. The confusing part is if you look under StudyVersion entity/class then it is studyTitle which may be where this stems from. I updated it to the scientificTitle attribute (from the entity of StudyProtocolDocumentVersion).

**Comment by ASL-rmarshall** [comment]
We agreed not to create this as a separate rule because the need for a scientific title will be checked as part of the generic cardinality checking.
